code/datums/mutantraces.dm

```diff
@@ -1111,6 +1111,10 @@ TYPEINFO(/datum/mutantrace)
 		if(ishuman(M))
 			M.mob_flags |= IS_BONEY
 			M.blood_id = "calcium"
+			M.head_tracker = M.organHolder.head
```

Comment:
We already have a head tracker with the `head` var on the skeleton mutantrace - could you rename that to `head_tracker` and just use that?